packages/frontend/src/components/claims/empty/styles.module.css

```diff
@@ -1,3 +1,19 @@
 .root {
-    @apply flex flex-col items-center gap-4 text-center max-w-screen-sm;
+    @apply flex items-center justify-center max-w-screen-sm h-[55vh];
+}
+
+.subContainer {
+    @apply flex flex-col items-center justify-center text-center gap-7 px-4;
+}
+
+.button {
+    @apply bg-gray-900 !px-12 !py-3 text-sm;
```

Comment:
Can't we use the button `size=small` here? I'd also avoid changing the bg to a slight darker color, we'd lose consistency across the application, if we have to update color for UI components (by design) we should do that in the UI component styles (same for paddings and text sizes).

Reply:
I will keep that in mind and avoid overriding styles.